extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Dialog.scss

```diff
@@ -37,3 +37,9 @@
 	margin: 5px auto 0 auto;
 }
 
+.insert-infobox-empty-state {
```

Comment:
I'm not sure if this is the best place for adding these styles. @inez do you have any suggestions?